Assets/MixedRealityToolkit/Sharing/README.md

```diff
@@ -133,7 +133,7 @@ Please sync to latest XTools
 ### [Plugins](Plugins)
 ---
 Contains compiled architecture specific binaries for SharingClient.dll which are required by the Unity application for accessing sharing APIs.
-Binaries are compiled from the native [MixedRealityToolkit\Sharing](https://github.com/Microsoft/MixedRealityToolkit/tree/master/Sharing).
+Binaries are compiled from the native [MixedRealityToolkit\Sharing](https://github.com/Microsoft/MixedRealityToolkit/tree/Dev_Working_Branch/Sharing).
```

Comment:
Why are we changing the branches everywhere? From master to Dev_Working_Branch?

Reply:
Because this will be targeting the dev branch and all links would be broken otherwise.
When this eventually gets merged to master, then the links should be updated to master.